__tests__/backend/taxjarApi.test.ts

```diff
@@ -0,0 +1,154 @@
+import _default from "taxjar/dist/util/request";
+import { createOrderTransaction, fetchTaxes } from "../../backend/taxjarApi";
+import { TaxJarConfig } from "../../backend/types";
+import { getTaxJarConfig } from "../../backend/utils";
+import {
+  dummyFetchTaxesPayload,
+  dummyFetchTaxesResponse,
+  dummyOrderCreatedPayload,
+  dummyTaxJarConfig,
+} from "../utils";
+import { TaxForOrderRes } from "taxjar/dist/util/types";
+
+jest.mock("taxjar/dist/util/request");
+
+const mockedTaxJarRequest = <jest.Mock>_default;
+
+describe("TaxJar API handlers", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("fetches taxes from taxjar", async () => {
+    const mockedTaxJarResponseData = dummyFetchTaxesResponse;
+    const post = jest.fn((url: any, params: any) => ({
+      mockedTaxJarResponseData,
+    }));
+    const mockedTaxJarResponse = mockedTaxJarRequest.mockImplementationOnce(
+      () => ({
+        post: post,
+      })
+    );
+
+    const payload = dummyFetchTaxesPayload;
+    const config: TaxJarConfig = dummyTaxJarConfig;
+
+    const response = await fetchTaxes(payload, config);
```

Comment:
I personally think mocking fetch requests is an anti-pattern and we should avoid it: https://kentcdodds.com/blog/stop-mocking-fetch

It doesn't have to be changed right now, but maybe we could use `msw` in future instead of mocking requests? https://github.com/mswjs/msw